File: engine/b/src/main/java/net/crazyinvestor/engine_b/dto/BithumbTickTypes.java
package net.crazyinvestor.engine_b.dto;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Getter
public enum BithumbTickTypes {
    MIN_30("30M"),
    HR_1("1H"),
    HR_12("12H"),
    HR_24("24H"),
    MID("MID"),
    ;

    private final String tickType;
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/enums/BithumbTickTypes.kt
package net.crazyinvestor.ticker_producer_engine.enums

enum class BithumbTickTypes(val tickType: String) {
    MIN_30("30M"),
    HR_1("1H"),
    HR_12("12H"),
    HR_24("24H"),
    MID("MID")
    ;
}

val TICK_TYPES = listOf("30M", "1H", "12H", "24H", "MID")